src/frontends/tensorflow/src/op_table.cpp

```diff
@@ -140,6 +142,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops() {
         {"Asinh", CreatorFunction(translate_unary_op<v3::Asinh>)},
         {"Atan", CreatorFunction(translate_unary_op<v0::Atan>)},
         {"Atanh", CreatorFunction(translate_unary_op<v3::Atanh>)},
+        {"Case", CreatorFunction(translate_case_op)},
```

Comment:
please add layer tests for this Operation

Reply:
@rkazants It would be greatly appreciated if you could guide me on the appropriate folder where I should add the layer tests for this operation. As I am new to open-source contributions, your guidance would be very helpful in ensuring I follow the correct structure and conventions for adding the test case for this operation.